_projects/2017/sunpy/roi.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Region of Interest
+desc: Meta-Object creation for holding temporal and spatial features.
+# add a description of your project
+requirements:
+ - Familiarity with Python
+ - Understanding of design patterns
+difficulty: High
+issues:
+ - https://github.com/sunpy/sunpy/issues/164
+mentors: 
+ - dpshelio
+ - wafels
+initiatives:
+ - GSOC
+tags:
+ - sunpy
+collaborating_projects:
+ - sunpy
+---
+
+#### Description
+
+This project aims to plan and implement a object that could be used to
+define a region of interest and interact with the rest of data types used
```

Comment:
I would describe more what you mean my ROI, i.e.: a region of interest, in any physical dimension, such as spatial area, temporal or spectral window.